spec/requests/incident_spec.rb

```diff
@@ -57,23 +57,23 @@ def check_for_question(choice, options = {})
 
       expect(current_path).to match(%r{\/involved_civilians(\/new)?})
       choose_for_question 'No', text: 'assaulted officer'
-      choose_for_question 'In custody', text: 'arrested and / or in custody'
+      choose_for_question 'In custody (other)', text: 'arrested and / or in custody'
       fill_in 'highest charge', with: 'Fake Charge 1234'
       choose_for_question 'Yes', text: 'perceived armed'
       check_for_question 'Other dangerous weapon', text: 'Type of weapon'
       choose_for_question 'Yes', text: 'confirmed armed'
       check_for_question 'Firearm', text: 'Weapons'
       check_for_question 'Rifle', text: 'Firearm types'
       choose_for_question 'Yes', text: 'resisted'
-      choose_for_question 'Assaultive', text: 'highest level of resistance'
+      choose_for_question 'Assaultive', text: 'highest level of active resistance'
       choose_for_question 'Yes', text: 'force used on this civilian'
       check 'Electronic control device'
       check 'Head'
       check 'Signs of alcohol impairment'
       choose_for_question 'Yes', text: 'injured'
       choose_for_question 'Serious bodily injury', text: 'Civilian injury level'
       check 'Abrasion/Laceration'
-      choose_for_question 'Admitted to hospital - critical injuries', text: 'aid received'
+      choose_for_question 'Admitted to hospital - critical injuries', text: 'aid received - '
```

Comment:
With the new question change, it's hard to tell which question this is referring to (and relying on a hyphen as a differentiator is a bit brittle I think). I'd just use the full question, "Civilian medical aid received - choose highest applicable". Yeah I know we're lazy in some of the other selectors, but they're pretty unambiguous in what question they refer to.